shelley/chain-and-ledger/executable-spec/test/Generator/Core/Constants.hs

```diff
@@ -127,3 +131,6 @@ minSlotTrace = 1000
 -- | Maximal slot for CHAIN trace generation.
 maxSlotTrace :: Int
 maxSlotTrace = 5000
+
+frequencyLowMaxEpoch :: Word64
```

Comment:
would you mind adding a comment to this?

Reply:
👍